src/lightning_app/utilities/tree.py

```diff
@@ -17,16 +17,6 @@ def breadth_first(root: "Component", types: Type["ComponentTuple"] = None):
     yield from _BreadthFirstVisitor(root, types)
 
 
-def depth_first(root: "Component", types: Type["ComponentTuple"] = None):
```

Comment:
reviewer: removed because depth_first is unused

Reply:
Yes.